app/models/bgs_dependents/veteran.rb

```diff
@@ -55,6 +57,19 @@ def veteran_response(participant, va_file_number, address, end_product)
 
     private
 
+    def user_veteran_attributes
+      external_key = @user.common_name || @user.email
```

Comment:
Forward-thinking, non-blocking comment: Do we want to move `external_key` to a method, or do we not foresee re-use of this logic?

Reply:
I'm glad you pointed this out. We don't even need the external key since we merged another branch that passed in the user object. Great shout out!